packages/core/src/media/controls/controls.ts

```diff
@@ -165,6 +181,9 @@ export const createControllerStore = <TElement extends HTMLMediaElement>(
         hasPlayed: false,
         playing: !element?.paused,
         fullscreen: false,
+        pictureInPicture: false,
```

Comment:
Let's make sure this value is being updated properly on enter/exit of PiP. Users can consume this value to conditionally render UI when PiP is active, so we need to make sure it toggles correctly. I can't test locally currently so just leaving this here to make sure it's tested